2021q4/kde.adoc

```diff
@@ -0,0 +1,38 @@
+
+
+* Just two CMake updates this quarter, ending up with CMake 3.22.1. Some more patches have landed upstream, and CMake is soon to switch to `share/man` for manpages on FreeBSD. When it does, there will be plenty of `pkg-plist` churn.
+* Monthly releases of KDE Frameworks, KDE Plasma, and KDE Gear kept the exp-runs going. kde@ would like to thank Antoine for overseeing our many exp-run requests. We are now at KDE Frameworks 5.89 (latest release as of December 2021), link:https://kde.org/announcements/plasma/5/5.23.4/[KDE Plasma Desktop 5.23.4] and link:https://kde.org/announcements/gear/21.12.0/[KDE Gear 21.12].
+* Qt 5 is not receiving any open source updates from the Qt Company, but the KDE Community maintains its own set of patches that backport many fixes from Qt 6. Work is underway to import the KDE patch collection.
+* Qt 6 remains tantalizingly close. There hasn't been real progress on the crash-on-exit problem, though.
+* *deskutils/kalendar* is a relatively new port that uses KDE technologies for a desktop (appointments) calendar.
+* *deskutils/latte-dock*, an alternative launcher for KDE Plasma (and other environments) was updated to each of its bugfix releases.
+* *devel/qbs* and *devel/qtcreator* were updated. Qbs (or "Qt Build System") is a declarative build system styled along the lines of declarative QML programs. (Note that Qbs is not used by Qt itself).
+* *graphics/digikam* was updated to the latest release and now supports both ImageMagick 6 and ImageMagick 7. Speaking of which, a new `USES=magick` was introduced to simplify ports that depend in ImageMagick.
+* *graphics/ksnip*, one of several screenshot-applications for KDE Plasma (and other environments) had a lots-of-bugfixes update.
+* *graphics/skanpage* is a new port that scans multiple pages and produces a PDF of the whole.
+* *multimedia/qt5-multimedia* now ignores gstreamer-gl (rather than implicitly building with it as a dependency if it is installed a build time).
+* *net-im/ruqola* is a Rocket Chat client, updated to the latest release.
+* *security/qtkeychain* has a new release.
+
+Elsewhere in the software stack, kde@ also maintains ports that support the desktop in general. Some highlights are:
+
+* *devel/libphonenumber* keeps chasing changes to the world's phone numbers (the FreeBSD foundation can be reached at +1.720.207.5142).
+* *graphics/poppler* updated this much-used PDF-rendering library.
+* *multimedia/pipewire*, the audio-and-video successor to pulseaudio, was updated and now supports SSL as well.
+* *net/py-pytradfri* got several updates so you can control your lights from FreeBSD.
+* *print/freetype2* was updated to the latest release; relatedly, there was am update to *x11-toolkits/libXft*.
+* *print/harfbuzz*, the text-shaping library, was updated for more font type support.
+* *sysutils/bsdisks* is an implementation of DBus interfaces for examining disks (drives, partitions, etc.). It is also used for removable-disk notifications.
```

Comment:
https://cgit.freebsd.org/ports/tree/sysutils/bsdisks/Makefile#n5 says otherwise. (Or did you mean "people part of kde@"?)